api/common/logger.go

```diff
@@ -119,14 +119,14 @@ func GetLogger(name string) *LogHandle {
 
 	if logger, ok := loggers[name]; ok {
 		if name != "main" && name != "fuse" {
-			logger.Level = cloudLogLevel
+			logger.SetLevel(cloudLogLevel)
```

Comment:
`SetLevel` calls `atomic.StoreUint32` which avoids the race:

https://github.com/sirupsen/logrus/blob/42baed85eb233bfc674562af0ef6e99c45f5f384/logger.go#L357